docker-compose.yml

```diff
@@ -29,6 +29,11 @@ services:
               - ra2.boulder
               - va2.boulder
               - publisher2.boulder
+        # Use dns-test-srv as a backup to Docker's DNS server. If there's a name
```

Comment:
Also: this comment is a bit confusing in general. I think it might be better to say something like "DNS server used by container for names not contained in /etc/hosts" since as far as I can tell from the docs there is no such thing as "Docker's DNS", if you remove this line it simply uses your hosts DNS settings (docs state it just copies the hosts `/etc/resolv.conf` if `--dns` isn't specified).